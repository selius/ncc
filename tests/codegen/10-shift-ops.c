int main()
{
	int a;
	int b;

	a = 3;
	b = 2;
	__print_int(a << b);
	__print_int(4 << b);
	__print_int(4 << 1);

	a = 28;
	b = 1;
	__print_int(a >> b);
	__print_int(a >> 2);
	__print_int(8 >> 3);
	
	return 0;
}
