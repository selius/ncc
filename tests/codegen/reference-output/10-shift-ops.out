12
16
8
14
7
1
